src/python/pants/core/goals/fmt.py

```diff
@@ -38,6 +41,14 @@ class FmtResult(EngineAwareReturnType):
     stderr: str
     formatter_name: str
 
+    def __post_init__(self):
+        log = f"Output from {self.formatter_name}"
+        if self.stdout:
+            log += f"\n{self.stdout}"
+        if self.stderr:
+            log += f"\n{self.stderr}"
+        logger.debug(log)
```

Comment:
Because the `level` of this `FmtResult` is `DEBUG`, its `message` field should already be logged when it completes, and this logging should probably be accomplished that way. 

There is a bug in this area right now though, which I hope to have patches out for today: see #14867.

Reply:
Is that true? the `level` property only uses `DEBUG` if the formatter was skipped (otherwise uses `WARN`/`INFO`)

Reply:
Ah, yea: you're right.